### PR TITLE
Fix typo and wording

### PR DESCRIPTION
**Pull request by cymen**
Minor cleanup to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ component.
 **Warning**
 
 Because falcor is intrinsically asynchronous your code can not rely on any one
-piece of state being present when rending. In the example above we give
+piece of state being present when rendoring. In the example above we give
```
Comment (ekosz) [inline README.md]: Is rendoring a word?

Comment (cymen) [inline README.md]: Oh, I think not. I think I typo'ed :). It should be `rendering`. I'll fix.

**Comment by cymen** [comment]
Updated `rendoring` -> `rendering`.

**Comment by ekosz** [comment]
Thanks for the PR @cymen !

**Pull request merged**